server/database/schemas/User.js

```diff
@@ -1,28 +1,40 @@
-const mongoose = require('mongoose');
-const { MongooseAutoIncrementID } = require('mongoose-auto-increment-reworked');
-const bcrypt = require('bcryptjs');
-const R = require('ramda');
+const mongoose = require("mongoose");
+const { MongooseAutoIncrementID } = require("mongoose-auto-increment-reworked");
+const bcrypt = require("bcryptjs");
+const R = require("ramda");
 
 const { Schema } = mongoose;
 
-const userSchema = new Schema({
-  username: { type: String, lowercase: true, required: true, unique: true, immutable: true },
-  username_case: { type: String, required: true },
-  password: { type: String, required: true },
-  profile_pic: { type: String },
-  first_name: { type: String, maxlength: 20 },
-  last_name: { type: String, maxlength: 20 },
-  bio: { type: String, maxlength: 240 },
-  created_at: { type: Date, default: Date.now, immutable: true },
-  updated_at: { type: Date },
-}, { versionKey: false });
-
-if (process.env.NODE_ENV !== 'test') {
-  MongooseAutoIncrementID.initialise('counters');
+const userSchema = new Schema(
+  {
+    username: {
+      type: String,
+      lowercase: true,
+      required: true,
+      unique: true,
+      immutable: true,
+    },
+    first_name: { type: String, maxlength: 20 },
+    last_name: { type: String, maxlength: 20 },
+    username_case: { type: String, required: false },
+    phone_number: { type: String, require: false },
+    birth_year: { type: Number, required: false },
+    email_address: { type: String, required: false },
+    password: { type: String, required: false },
+    forest: { type: String, require: false },
```

Comment:
what will be here? the id of the forest? if so, let's change the name to "forest_id"